kahuna/public/js/edits/image-editor.js

```diff
@@ -75,6 +77,16 @@ imageEditor.controller('ImageEditorCtrl',
 
 
     ctrl.usageRights = imageService(ctrl.image).usageRights;
+
+    function updateUsageRightsCategory() {
+        let category = ctrl.categories.find(cat => cat.value === ctrl.usageRights.data.category);
+        ctrl.usageRightsCategory = category && category.name;
+        ctrl.showUsageRights = ctrl.usageRightsCategory === undefined;
```

Comment:
In theory, yes. However, `None` isn't an option...

![screen shot 2015-08-21 at 14 22 39](https://cloud.githubusercontent.com/assets/836140/9409151/2f1c1790-4810-11e5-96c4-cb13ce1377df.png)